source/playbooks/playbook-permissions.rst

```diff
@@ -0,0 +1,74 @@
+Roles and Permissions
+=====================
+
+There are different ways to access and interact with playbooks and this is controlled in the System Console in the form of permissions. Permissions can be granted in a variety of ways, to allow for different combinations of access and visibility, and to different roles. Two permission schemes are provided in Mattermost:
+
+* **System Scheme:** Applies permissions universally across all teams and channels.
+* **Team Override Schemes:** Allow admins to customize permissions for each team (available in Mattermost Enterprise and Professional).
+
+You can set the default permissions granted to System Admins, Team Admins, Channel Admins, Playbook Admins, Guests (if enabled), and All Members. The permissions granted in the System Scheme apply system-wide, meaning:
+
+* **Guests:** If Guest Accounts are enabled, permissions apply to guest users in all channels, in all teams.
+* **All Members:** Permissions apply to all members, including Admins, in all channels, in all teams.
+* **Channel Administrators:** Permissions apply to all Channel Admins in all channels, in all teams.
+* **Playbook Admins:** Permissions apply to all Playbook Admins in all channels, in all teams.
+* **Team Administrators:** Permissions apply to all Team Admins, in all teams.
+
+For more information about System and Team Override Schemes, refer to the `Advanced Permissions <https://docs.mattermost.com/onboard/advanced-permissions.html>`__ documentation.
+
+Roles
+-----
+
+**Member**
+
+In the context of Playbooks, members are users of Mattermost who belong to the team within which the run is active and who may have a vested interest in the run. Depending on the permissions granted, members can have the same privileges as Playbook Admins or restricted privileges. Members are also referred to as participants in a run.
```

Comment:
Members are not the same as participants. Playbooks have members runs have participants.